docs/release-notes.md

```diff
@@ -11,6 +11,9 @@ sidebarDepth: 1
 # Release Notes
 
 ## 14.0.0
+### CI/CD Action Items for Right Sizing
+* Users will start to see Action Items for right sizing their resources
```

Comment:
Action Items for right-sizing workloads are now available in CI/CD. You'll see one Action Item per cluster where the workload appears.

Reply:
What Robert said ^^